### ISSUE TITLE
Change URL for model implementation principles?

### ISSUE THREAD

**Issue by hfrick** [tidymodels/tidymodels.org-legacy#240]
We could move https://tidymodels.github.io/model-implementation-principles/ to, e.g., `https://tidymodels.org/model-implementation-principles/`

**Comment by juliasilge** [comment]
Just as a `_redirects`, you mean?

**Comment by hfrick** [comment]
probably, yes? I actually don't really know so whatever makes sense!

**Comment by juliasilge** [comment]
I don't think we could move the whole thing easily (since it is a bookdown project in a different repo) but we could [redirect](https://moz.com/learn/seo/redirection), if we think the URL is a problem.

**Comment by hfrick** [comment]
ah, that makes sense, thanks! I wouldn't say the URL is a problem but maybe a nice to have.
